src/App.js

```diff
@@ -1,12 +1,11 @@
-//import logo from './logo.svg';
 import './App.css';
+import CreateTabs from "./CreateTabs.js"
 
 function App() {
```

Comment:
Let's use an arrow function for this component (should be in the React tutorial)

Reply:
Boom.  Done son